beprof/example.py

```diff
@@ -0,0 +1,15 @@
+import urllib.request
+from beprof.curve import Curve
+import numpy as np
+
+url = 'http://berkeleyearth.lbl.gov/auto/Global/Complete_TAVG_complete.txt'
```

Comment:
OK I will find better example. By rising and falling you mean it should be unimodal, or it can have many extremums?